_features/discoverable.html

```diff
@@ -2,34 +2,22 @@
 title: Discoverable
 slug: discoverable
 description: Are identifiable as applications and search engines can find them.
+usecases:
+- <p>Progressive Web Applications offer unique capabilities; as a developer, I want
+  my web applications to be promoted in search engines when the user is using
+  modern platforms.</p>
+- <p>As a search engine owner I want to highlight those applications with best UX
+  promoting them to the upper positions.</p>
+- <p>As a browser developer I want to give apps special capabilities and allow
+  them to <em>escape the tab</em>.</p>
+- <p>As a marketplace owner I want to gather information about progressive web
+  application to rating and sorting them.</p>
 technologies:
 - web-manifest
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
-<!--
-<h2>Use cases</h2>
-<ul class="use-cases">
-  <li>As a user, I want to revisit a site and get its contents even if no network is available.</li>
-  <li>As a user, I want to browse any kind of content I visited at least once, even under situations of poor connectivity.</li>
-  <li>As a developer, in situations with no connectivity, I want to control what is shown to the user.</li>
-</ul>
-<h2>Technologies</h2>
-<ul class="grid">
-  <li>
-    <a href="../technologies/service-workers.html">
-      <h3 id="Service Workers">Service Workers</h3>
-      <p>Cache network resources and enable <strong>super fast</strong> access to them.</p>
-    </a>
-  </li>
-  <li>
-    <h3 id="App Shell">App Shell</h3>
-    <p>Use app shell architecture to reach almost instant first draw.</p>
-  </li>
-</ul>
-<h2>Resources</h2>
-<ul class="resources">
-  <li><a href="https://github.com/GoogleChrome/sw-precache" target="_blank">sw-precache</a> is a node module to generate service worker code that will precache specific resources.</li>
-  <li><a href="https://github.com/mozilla/oghliner" target="_blank">oghliner</a> is not only a template but a tool for deploying Offline Web Apps to GitHub Pages.</li>
-  <li><a href="https://www.talater.com/upup/" target="_blank">UpUp</a>is a tiny script that makes sure your site is always there for your users.</li>
-</ul>
--->
+<p>As soon as you start writing progressive web applications you want the World to
+  know about them. If a user visits your application in a modern platform, all your
+  web site's progressive super-powers simply emerege but, <em>what if the user
+  never heard about you?</em> Progressive Web Applications are discoverable
+  because they allow search engines and Web indices to find your application and
+  highlight their capabilities.</p>
```

Comment:
> Note also Indexes vs. indices, which does a good job comparing the two words, and Grammar: Indexes vs. Indices, which argues for the latter to avoid confusion with the verb indexes.
I use _indices_ if there is a mathematical, scientific or formal context. I preferred to change it here because I don't want the tone of the text to be too formal.